src/btreeset.rs

```diff
@@ -314,6 +326,9 @@ where
     /// Returns the first key in the set. This key
```

Comment:
Seems we did not add the complexity for `clear` above.

Reply:
Good catch. Updated, though I may be wrong. My reasoning is `fn clear_new()` takes O(n) time because clearing the allocator: The self.allocator.clear() operation deallocates all memory used by the map, which involves traversing all allocated chunks. This operation is proportional to the number of elements in the map (O(n)).

Reply:
Yeah seems reasonable I would also make a similar argument.